src/Input.cpp

```diff
@@ -140,6 +140,7 @@ static JoystickInfo loadJoystick(SDL_Joystick *joystick, IniConfig *config)
 	for (size_t i = 0; i < state.axes.size(); i++) {
 		std::string axisName = "Axis" + std::to_string(i);
 		if (!config->HasEntry(joystickName, axisName)) {
+			state.axes[i].deadzone = 0.2;
```

Comment:
I'd prefer if this deadzone is set to `0.05` or similar, a 20% deadzone is extremely aggressive for anything that's not a thumbstick and the OS usually has a pretty large deadzone for most hardware. A 5% deadzone will filter out issues with sticks being almost centered but still reporting a non-zero value, and be a sane default for users who do not know about the ability to change deadzones in the config file.

Reply:
Done! And ready to merge!